Webhook management (#683)
mvisonneau/gitlab-ci-pipelines-exporter: docs/configuration_syntax.md

    # environment variable)
    secret_token: 063f51ec-09a4-11eb-adc1-0242ac120002

>     add_webhooks: 

Reviewer: ```suggestion
    manage_projects:
```

      # Whether to add webhooks to projects from wildcards when
      # exporter starts (optional, default: false)
      on_init: false
      # Whether to add webhooks to projects from wildcards
      # on a regular basis (optional, default: false)
      scheduled: false
      # Interval in seconds to add webhooks to projects 
      # from wildcards (optional, default: 43200)
      interval_seconds: 43200

>     # GCPE Webhook endpoint URL 
>     webhook_url: https://gcpe.example.net/webhook

Reviewer: ```suggestion
      # GCPE Webhook endpoint URL accessible from GitLab
      advertised_url: https://gcpe.example.net/webhook
```

>     # Whether to remove webhooks when GCPE shutsdown
>     remove_webhooks: false

Reviewer: ```suggestion
      # Whether or not to remove webhooks when GCPE shuts down
      remove_on_delete: false
```
